controllers/spacebindingrequest/spacebindingrequest_controller.go

```diff
@@ -260,6 +266,18 @@ func (r *Reconciler) ensureSpaceBinding(ctx context.Context, memberCluster clust
 	return r.updateExistingSpaceBinding(ctx, spaceBindingRequest, spaceBinding)
 }
 
+// sbrOwnsSpaceBinding check if the SpaceBinding was created by the given SpaceBindingRequest resource
+// returns true if it was created by the SBR
+// returns false if it wasn't created by the SBR
+func sbrOwnsSpaceBinding(spaceBinding *toolchainv1alpha1.SpaceBinding, spaceBindingRequest *toolchainv1alpha1.SpaceBindingRequest) bool {
+	sbrLabel := spaceBinding.Labels[toolchainv1alpha1.SpaceBindingRequestLabelKey]
+	sbrNamespaceLabel := spaceBinding.Labels[toolchainv1alpha1.SpaceBindingRequestNamespaceLabelKey]
+	if sbrLabel != spaceBindingRequest.GetName() || sbrNamespaceLabel != spaceBindingRequest.GetNamespace() {
+		return false
+	}
+	return true
```

Comment:
```suggestion
	return sbrLabel == spaceBindingRequest.GetName() && sbrNamespaceLabel == spaceBindingRequest.GetNamespace()
```

Reply:
indeed this is shorter/nicer! thanks a lot!